waku/v2/node/wakunode2.nim

```diff
@@ -814,7 +818,11 @@ proc start*(node: WakuNode) {.async.} =
   # TODO Get this from WakuNode obj
   let peerInfo = node.peerInfo
   info "PeerInfo", peerId = peerInfo.peerId, addrs = peerInfo.addrs
-  let listenStr = $peerInfo.addrs[^1] & "/p2p/" & $peerInfo.peerId
+  var listenStr = ""
+  for address in node.announcedAddress:
+    var fulladdr = "[" & $address & "/p2p/" & $peerInfo.peerId & "]" 
+    addf(listenStr, fulladdr)
```

Comment:
Not sure I see the value of using `addf` rather than simply `&=` if we don't apply any particular formatting? It might just be because I don't know the `addf` function very well. :)

Reply:
fixed in 5eff5597eabec4afaca39814bcd04d751c41ed2f